index.html

```diff
@@ -1,251 +1,505 @@
-            </p>
-            <div class="box-container">
-              <ul class="d-flex font-inter">
-                <li class="box">html</li>
-                <li class="box">bootstrap</li>
-                <li class="box">Ruby</li>
-              </ul>
+      <div class="grid-container">
+        <div class="grid-items d-flex grid-item-1">
+          <div class="grid-item-card text-white">
+            <div class="grid-item-card-content mobile-item-card">
+              <h2 class="font-crete-round">Profesional Art Printing Data</h2>
+              <p class="font-inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="box-container d-flex">
+                <ul class="d-flex font-inter list-none">
+                  <li class="box">html</li>
+                  <li class="box">bootstrap</li>
+                  <li class="box">Ruby</li>
+                </ul>
+              </div>
             </div>
+            <button class="primary-button primary-color font-inter">
+              See Project
+            </button>
           </div>
-          <button class="primary-button primary-color font-inter">See Project</button>
         </div>
-      </div>
-      <div class="grid-items d-flex">
-        <div class="grid-item-card text-white">
-          <div class="grid-item-card-content">
-            <h2 class="font-crete-round">Profesional Art Printing Data</h2>
-            <p class="font-inter">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-            </p>
-            <div class="box-container">
-              <ul class="d-flex font-inter">
-                <li class="box">html</li>
-                <li class="box">bootstrap</li>
-                <li class="box">Ruby</li>
-              </ul>
+        <div class="grid-items d-flex grid-item-2">
+          <div class="grid-item-card text-white mobile-item-card">
+            <div class="grid-item-card-content">
+              <h2 class="font-crete-round">Profesional Art Printing Data</h2>
+              <p class="font-inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="box-container">
+                <ul class="d-flex font-inter list-none">
+                  <li class="box">html</li>
+                  <li class="box">bootstrap</li>
+                  <li class="box">Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <button class="primary-button primary-color font-inter">
+              See Project
+            </button>
+          </div>
```

Comment:
- [ ] When I hover over each of the 6 smaller project sections, the the white text and its dark opacity should disappear and a `See project` button appears. Ensure that the See Project button is designed according to the Figma design and it redirects a user to the project live demo or your GitHub repo.